soh/src/code/z_actor.c

```diff
@@ -2506,7 +2507,7 @@ void Actor_UpdateAll(PlayState* play, ActorContext* actorCtx) {
         refActor = &GET_PLAYER(play)->actor;
         KREG(0) = 0;
         Actor_Spawn(&play->actorCtx, play, ACTOR_EN_CLEAR_TAG, refActor->world.pos.x,
-                    refActor->world.pos.y + 100.0f, refActor->world.pos.z, 0, 0, 0, 1);
```

Comment:
All the .cpp files have to use 1/0 because true/false mean different things in C++ compared to the true/false being a define to 1/0 in C files. I just checked to see and it looks like the one in z_actor.c outlined above is the only one I somehow missed, so I'll change that one real quick.